config/yurt-app-manager/samples/nodepool/apps_v1alpha1_nodepool_cloud_test.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: apps.openyurt.io/v1alpha1
```

Comment:
change file name from  'apps_v1alpha1_nodepool_cloud_test.yaml ' to 'apps_v1alpha1_nodepool_cloud.yaml'